backend/app/presenter/collection_presenter.rb

```diff
@@ -3,8 +3,9 @@
 class CollectionPresenter
```

Comment:
`nit`

Not sure about this naming. 🤔 I mean... I guess it is technically a presenter. IMO it should be named as CollectionFormatter instead. But this is a verrrry `nit` comment